packages/builder/src/components/automation/AutomationBuilder/FlowChart/FlowItemHeader.svelte

```diff
@@ -123,4 +215,47 @@
     display: flex;
     align-items: center;
   }
+
+  .hide-context-actions {
+    display: none;
+  }
+  input {
+    font-family: var(--font-sans);
+    color: var(--ink);
+    background-color: transparent;
+    border: none;
+    font-size: var(--spectrum-alias-font-size-default);
+    width: 260px;
+    box-sizing: border-box;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
```

Comment:
It might be nice to add top-left and top-right border radius to the input, to match the actual header, so that it doesn't get squared edges when typing.